.github/workflows/cicd.yml

```
@@ -57,16 +62,17 @@ jobs:
     name: Build package distribution
     steps:
       - uses: actions/checkout@v1
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)